.github/workflows/spelling.yml

```
@@ -0,0 +1,54 @@
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  spelling:
+    name: Spell checking
+    permissions:
+      contents: read
+      pull-requests: read
+      checks: write
+      comments: write
```

actionlint (1.7.12)

[error] unknown permission scope "comments". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)